kf_lib_data_ingest/common/concept_schema.py

```diff
@@ -5,11 +5,11 @@
 
 
 class FileMixin(object):
-    ETAG = None
     SIZE = None
-    DATA_TYPE = None
+    URL = None
```

Comment:
I included URL and HASH, HASH_TYPE in the FileMixin and then URL_LIST and HASH_DICT in the GENOMIC_FILE concept because we shouldn't only cater to the types/structure of the Dataservice. This way we can use URL for a file path or file url and we can use URL_LIST for a list of filepaths or urls. Sort of similar for hashes

Reply:
I don't think that makes any sense.

First, you don't have any way to convert URL into URL_LIST or HASH into HASH_DICT, which means that all of your URL and HASH values are currently being thrown away.

> I thought we shouldn't only cater to the types/structure of the Dataservice

It has nothing to do with the dataservice. A file may always be stored in multiple locations and definitely always has infinitely many hash values, no matter where you choose to store the information about your files.

Reply:
Ok, so then if we only have URL_LIST and HASH_DICT, then what should go in here:
https://github.com/kids-first/kf-lib-data-ingest/blob/daf0a280dd43c46a8da6368355cf3e5de65d2883/tests/data/test_study/extract_configs/simple_tsv_example2.py#L39

Should it the mapping function still return a string even though will be mapped to something called `CONCEPT.GENOMIC_FILE.URL_LIST`?

Reply:
return a list